File: pizzayolo/src/main/java/pizzayolo/entity/Livraison.java
package pizzayolo.entity;

import java.time.LocalTime;

import javax.persistence.Column;
import javax.persistence.DiscriminatorValue;
import javax.persistence.Entity;

import com.fasterxml.jackson.annotation.JsonView;

@Entity
@DiscriminatorValue("L")
public class Livraison extends Commande {

	@Column(name = "heure_souhaite")
	@JsonView(JsonViews.CommandeWithItem.class)
	private String heureSouhaite;

	public Livraison() {
	}

	public String getHeureSouhaite() {
		return heureSouhaite;
	}

	public void setHeureSouhaite(String localTime) {
		this.heureSouhaite = localTime;
	}
}

File: pizzayoloangular/src/app/model/livraison.ts
import { Commande } from './commande';

export class Livraison extends Commande {
  public constructor(private _heureSouhaite?: string) {
    super();
  }
  get heureSouhaite(): string | undefined {
    return this._heureSouhaite;
  }

  set heureSouhaite(value: string | undefined) {
    this._heureSouhaite = value;
  }
}
